monitoring/heimdall_monitoring.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+#############################################################################
+#									    #
+#      This script does 						    #
+#            1: Enable prometheus for Heimdall.                             #
+#            2: setting up Prometheus and Grafana locally.                  #
+#            3: Create Grafana Dashboard for Heimdall.		            #
+#									    #
+#############################################################################
+
+## path of Grafana datasorce and dashboard json's
+json_path=$(pwd)
+echo $json_path
+
+## Enabling Prometheus for Heimdall
+sudo sed -i 's/prometheus = false/prometheus = true/' /home/ubuntu/.heimdalld/config/config.toml
```

Comment:
Can we use the home directory variable `$HOME` or something instead of using `home/ubuntu`?

Reply:
@jdkanani ya we can use **$HOME** or **~/** so, what do you say ?

Reply:
@ramakrishna183 Sounds good.